src/operators/rasterstacker.md

```diff
@@ -9,7 +9,23 @@ All inputs must have the same data type and spatial reference.
 
 ## Parameters
 
-_no parameters_
+| Parameter   | Type          | Description                                                 | Example Value                                |
+| ----------- | ------------- | ----------------------------------------------------------- | -------------------------------------------- |
+| renameBands | `RenameBands` | Specification of how to rename the bands to avoid conflicts | <pre><code>{ "type": "defaultSuffix" }</pre> |
+
+## Types
+
+The following describes the types used in the parameters.
+
+### RenameBands
+
+The `RenameBands` type is used to specify how to rename the bands to avoid conflicts.
+
+| Value                                    | Description                                                                          |
+| ---------------------------------------- | ------------------------------------------------------------------------------------ |
+| `{"type": "defaultSuffix"}`              | Appends "(duplicate)" to the band name until the conflict is resolved                |
+| `{"type": "suffix", "values": [string]}` | Specifies a suffix for all but the first input, to be appended in case of a conflict |
```

Comment:
i think you should be able to set a suffix for the first input. Maybe allow it to be empty?